servers/dsub/jobs/controllers/utils/generator.py

```diff
@@ -0,0 +1,156 @@
+import datetime
```

Comment:
nit: generator is too general, I would name this file jobs_generator or something similar

Reply:
Renamed to jobs_generator